### PR TITLE
Sf 735

### PR DESCRIPTION
**Pull request by shashank-naik**
Hey @chrisferagotti , can you please review this module. Any suggestions/tips would be helpful.

Thanks!

### CODE REVIEW

**Comment by chrisferagotti** [comment]
At first glance everything looks okay, but I haven't tested it. I can take a deeper look a little later. Is there anything in particular you had in mind? I mean, as far as suggestions/tips, was there any issue with it, or just looking for any kind of general optimizations/cleanup?

**Comment by shashank-naik** [comment]
It can be all of those. You can note all of your observations in there. They could range from security vulnerabilities to optimizations to even ill-named variables ($test1).  Basically, rip into it ☺.

If you want to test it, I have it up on dev. However, I would not spend too much time on testing it because we have a QA team to do it.

File: smokefree/sites/all/modules/custom/icfi_motion_ai_api/icfi_motion_ai_api.module [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  }
+}
+
+/**
+ * Check if the supplied API key is valid.
+ * Please see README.txt file of icfi_api_keys module.
+ *
+ * @return bool TRUE is the key matches the one in the database, FALSE otherwise.
+ */
+function is_valid_key() {
```
Comment (chrisferagotti) [inline smokefree/sites/all/modules/custom/icfi_motion_ai_api/icfi_motion_ai_api.module]: Three of the functions start with underscores; I know this is the old way to help denote public/private functions before PHP supported it, and I use it myself, I'm just not familiar enough with the way the rest of the modules associated with this one will work to know if the underscored/not-underscored are accurate. Could you verify?

Comment (chrisferagotti) [reply smokefree/sites/all/modules/custom/icfi_motion_ai_api/icfi_motion_ai_api.module]: Right. It didn't change the actual scope, but it was just used as a hint to devs. And I was just being overly picky. From what I can tell, this looks to me.

**Pull request merged**